devsetup/scripts/standalone.sh

```diff
@@ -180,6 +202,18 @@ scp $SSH_OPT $HOME/.ssh/id_ecdsa.pub root@$IP:/root/.ssh/id_ecdsa.pub
 if [[ -f $HOME/containers-prepare-parameters.yaml ]]; then
     scp $SSH_OPT $HOME/containers-prepare-parameters.yaml root@$IP:/root/containers-prepare-parameters.yaml
 fi
+# For multi-stack deployment, get extracted data from main node index 0 and copy to the target node
+if [[ "$EDPM_COMPUTE_SUFFIX" != "0"  ]]; then
```

Comment:
these conditional blocks will be moved into different make standalone_stack targets